mysql-test/suite/tokudb/t/dir_per_db_rename_with_non_alnum_chars.test

```diff
@@ -0,0 +1,22 @@
+--source include/have_tokudb.inc
```

Comment:
I'd avoid starting a new testcase and just add an
--echo #
--echo # TDB-72: blah blah
--echo #

block to the existing dir_per_db testcase